internal/mining/mining.go

```diff
@@ -1652,6 +1871,29 @@ nextPriorityQueueItem:
 		}
 	}
 
+	// If we did not already add auto revocations above, attempt to add them now.
+	// This case will be hit if the priority queue contained only vote
+	// transactions.
+	if isAutoRevocationsEnabled && !addedAutoRevocations {
```

Comment:
If only votes we would never get the `!isSSGen` condition?

Reply:
Correct, that case is handled by [this block](https://github.com/decred/dcrd/blob/0b9760fe6f42384e8e5fb0023e56fce83eb3be7e/internal/mining/mining.go#L1874-L1895) below.